app/src/main/java/com/android/streetworkapp/model/park/ParkRepositoryFirestore.kt

```diff
@@ -311,6 +311,30 @@ class ParkRepositoryFirestore(private val db: FirebaseFirestore, testing: Boolea
     }
   }
 
```

Comment:
Suggestion: you should also implement a function to delete the deleted user's events from a park

Reply:
Good catch, I have made this change! It consists of saving the list of parks deleted when using the event viewmodel and then passing this list to the park viewmodel in order to delete them from all parks. I'll let you have a look at this addition and let me know what you think of it.